core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -318,6 +324,20 @@
             exceptionLoggingMode("annotatedServiceExceptionVerbosity",
                                  DEFAULT_ANNOTATED_SERVICE_EXCEPTION_VERBOSITY);
 
+    private static final List<Predicate<InetAddress>> PREFERRED_IP_V4_CIDR =
+            CSV_SPLITTER.splitToList(getNormalized("preferredIpV4Cidr", "", unused -> true))
```

Comment:
- How about removing `Cidr` from the flag name and accepting both exact IP address and CIDR?
- Not exactly related with this PR, but we could improve `InetAddressPredicated.ofCidr()` to accept an exact IP address if `/` is not found in the string.

Reply:
That's a good idea. Fixed. 😄